### PR TITLE
Update gen3 lite and gripper macros

### PR DESCRIPTION
**Pull request by dyackzan**

### CODE REVIEW

File: kortex_description/grippers/robotiq_2f_85/urdf/robotiq_2f_85_macro.xacro [hunk 1]
```diff
@@ -25,9 +23,7 @@
         use_fake_hardware="${use_fake_hardware}"
         fake_sensor_commands="${fake_sensor_commands}"
         sim_ignition="${sim_ignition}"
-        sim_isaac="${sim_isaac}"
-        isaac_joint_commands="${isaac_joint_commands}"
-        isaac_joint_states="${isaac_joint_states}">
+        sim_isaac="${sim_isaac}">
```
Comment (MarqRazz) [inline kortex_description/grippers/robotiq_2f_85/urdf/robotiq_2f_85_macro.xacro]: [This xacro](https://github.com/PickNikRobotics/ros2_robotiq_gripper/blob/main/robotiq_description/urdf/robotiq_2f_85_macro.urdf.xacro#L9-L11) supports these parameters why are they being removed?

File: kortex_description/robots/kortex_robot.xacro [hunk 2]
```diff
@@ -84,9 +84,7 @@
         use_fake_hardware="${use_fake_hardware}"
         fake_sensor_commands="${fake_sensor_commands}"
         sim_ignition="${sim_ignition}"
-        sim_isaac="${sim_isaac}"
-        isaac_joint_commands="${isaac_joint_commands}"
-        isaac_joint_states="${isaac_joint_states}"/>
```
Comment (MarqRazz) [inline kortex_description/robots/kortex_robot.xacro]: Why are these removed?

File: kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro
Comment (MarqRazz) [inline kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro:1]: You are going to need to create a ros2_control.xacro ([like this one](https://github.com/PickNikRobotics/ros2_robotiq_gripper/blob/main/robotiq_description/urdf/2f_85.ros2_control.xacro)) for this gripper if you want it to work with `mock_hardware` or `Gazebo/Isaac`.

File: kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+  <xacro:macro name="load_gripper" params="
+    parent
+    prefix
+    use_fake_hardware:=false
+    fake_sensor_commands:=false
+    sim_ignition:=false
+    sim_isaac:=false
+    isaac_joint_commands:=/isaac_joint_commands
+    isaac_joint_states:=/isaac_joint_states
+    use_internal_bus_gripper_comm:=true">
```
Comment (MarqRazz) [inline kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro:13]: These are not needed if we don't want to spend the time adding the ros2_control.xacro.

File: kortex_description/grippers/robotiq_2f_85/urdf/robotiq_2f_85_macro.xacro (hunk 1 above)
Comment (dyackzan) [reply kortex_description/grippers/robotiq_2f_85/urdf/robotiq_2f_85_macro.xacro]: Reverted

File: kortex_description/robots/kortex_robot.xacro (hunk 2 above)
Comment (dyackzan) [reply kortex_description/robots/kortex_robot.xacro]: Reverted

File: kortex_description/arms/gen3_lite/6dof/urdf/kortex.ros2_control.xacro [hunk 4]
```diff
@@ -43,104 +54,75 @@
...
           <param name="max">2.69</param>
         </command_interface>
-        <command_interface name="velocity">
-          <param name="min">-1.6</param>
-          <param name="max">1.6</param>
-        </command_interface>
         <state_interface name="position">
           <param name="initial_value">${initial_positions['joint_1']}</param>
         </state_interface>
-        <state_interface name="velocity"/>
```
Comment (MarqRazz) [inline kortex_description/arms/gen3_lite/6dof/urdf/kortex.ros2_control.xacro]: Sorry only the command `velocity` interface needed to be removed.

Comment (dyackzan) [reply kortex_description/arms/gen3_lite/6dof/urdf/kortex.ros2_control.xacro]: Ok, added those back in

File: kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro
Comment (dyackzan) [reply kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro:1]: Created an issue for this: https://github.com/Kinovarobotics/ros2_kortex/issues/193

File: kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro (hunk 3 above)
Comment (dyackzan) [reply kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro:13]: Should I remove these? They might be helpful in the future once the ros2_control.xacro is created and I don't think they hurt anything by being there for now

File: kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro
Comment (MarqRazz) [reply kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro:1]: Thanks for creating the issue.

File: kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro (hunk 3 above)
Comment (MarqRazz) [reply kortex_description/grippers/gen3_lite_2f/urdf/gen3_lite_2f_macro.xacro:13]: It's fine to leave them.

**Review by MarqRazz — APPROVED** [review]
This is close enough. I will patch this up in a follow on PR.
